### DISCUSSION TITLE
How to connect to snipe IT through tailscale?

### DISCUSSION THREAD

**Discussion by zzzzMinde** [snipe/snipe-it#13601] [Q&A]
Hi all,

I'm currently self hosting Snipe IT and am trying to see if I can connect to it externally without any port forwarding etc. Currently it is on a laptop with the IP of 192.168.1.139 and I'm able to edit host files and connect to it through a domain. It works the same for me if im changing the apache2 config / snipe .env config.

However, trying to connect through tailscale and it redirects me to a local ip / the domain and thus ruins it, and I cant figure out why it does that? If im hosting a different webserver on the same laptop different port, it seems to connect fine because its not redirecting my tailscale IPV4 address and thus am able to access it externally without any problems, so why does snipe IT redirect me?

my .htaccess files seem like they dont redirect me to the APP_URL so why is it happening and how can i disable it so i can access snipe it through tailscale??

**Comment by zzzzMinde** [comment]
Ive come to the conclusion its Snipes internal redirection, getting rid of the apache VHOSTS leads me to connect to the apache server just fine

**Reply by Dusanst016** [reply]
Hi!
I am having the same issue. Could you please explain how did you go about getting rid of the apache VHOSTS?

**Reply by zzzzMinde** [reply]
Hi dusant, I do not have access to the system for the next week so when I’m able to access it again I’ll let you know what I have set up

**Reply by Dusanst016** [reply]
Thank you sir. I really do appreciate it.
